src/js/base/js-numbers.js

```diff
@@ -1257,11 +1257,14 @@ define("pyret-base/js/js-numbers", function() {
     function(m, n) {
       return m / n;
     },
-    function(m, n) {
+    function(m, n, errbacks) {
```

Comment:
E.g. this change is good, but then `_integerEquals` below  should have 3-arg functions